docs/modules/ROOT/nav.adoc

```diff
@@ -1,53 +1,80 @@
-* xref:native-memory.adoc[Native Memory]
-* xref:jvm-parameters.adoc[Configuring JVM Parameters]
-* xref:env-vars.adoc[Configuring Environment Variables]
-* xref:resource-configuration.adoc[Configuring Resource Limits]
-* xref:hazelcast-parameters.adoc[Configuring System Properties]
-* xref:advanced-networking.adoc[Advanced Networking]
-* xref:tls.adoc[Configuring TLS]
-* xref:user-code-namespaces.adoc[User Code Namespaces]
-* xref:authorization.adoc[Authorization Methods to Access Cloud Storage]
-* Data Pipelines
-** xref:jet-engine-configuration.adoc[Configuring the Jet Engine]
-** xref:jet-job-configuration.adoc[Running Data Pipelines]
-** xref:jet-job-snapshot.adoc[Saving the State of Data Pipelines]
-* xref:custom-config.adoc[Custom Configuration of Hazelcast cluster]
-* Management Center
-** xref:management-center-clusters.adoc[Hazelcast Clusters]
-** xref:management-center-external-access.adoc[External Access]
-** xref:management-center-persistence.adoc[Persistence]
-** xref:management-center-jvm-args.adoc[JVM Arguments]
-** xref:management-center-ldap.adoc[LDAP Security Provider]
-* xref:flow.adoc[Flow]
+* xref:flow.adoc[Deploy Flow]
+* xref:air-gapped-env.adoc[Run in air-gapped environments]
+* xref:scaling-upgrading.adoc[Scale and upgrade]
+
+.Configure
+// Configuration options
+* xref:connect-outside-kubernetes.adoc[Connect to Hazelcast from outside Kubernetes]
+* xref:scheduling-configuration.adoc[Assign Pods to nodes]
+
+* Memory & storage
+** xref:jvm-parameters.adoc[JVM parameters]
+** xref:native-memory.adoc[High-density memory store]
+** xref:tiered-storage.adoc[Tiered storage]
+
+* xref:backup-restore.adoc[Fault tolerance]
+
+* Data
+** Data structures
+*** xref:map-configuration.adoc[Map]
+*** xref:multimap-configuration.adoc[MultiMap]
+*** xref:topic-configuration.adoc[Topic]
+*** xref:queue-configuration.adoc[Queue]
+*** xref:cache-configuration.adoc[Cache]
+*** xref:vector-collection-configuration.adoc[Vector collection]
+*** xref:replicatedmap-configuration.adoc[ReplicatedMap]
+
+** Data pipelines
+*** xref:jet-engine-configuration.adoc[Configure the Jet Engine]
+*** xref:jet-job-configuration.adoc[Run data pipelines]
+*** xref:jet-job-snapshot.adoc[Save state of data pipelines]
+
+** xref:serialization-configuration.adoc[Data serialization]
+** xref:cp-subsystem.adoc[Enable CP subsystem]
+
+* Resilience
+** Partition groups & networking
+*** xref:high-availability-mode.adoc[High Availability Mode]
+*** xref:advanced-networking.adoc[Advanced network configuration]
+
+** Synchronize data across clusters
+*** xref:wan-replication.adoc[Configure WAN replication]
+*** xref:wan-sync.adoc[Synchronize WAN replicated data]
+
+** Security
+*** xref:tls.adoc[Configure TLS]
+*** xref:authorization.adoc[Authorization Methods to Access Cloud Storage]
+
+** Compute
+*** xref:user-code-deployment.adoc[User Code Deployment]
+*** xref:user-code-namespaces.adoc[User Code Namespaces]
+
+** Config properties
```

Comment:
I would rename `Config properties` as `Configuration Options`

Reply:
Since the category is 'Configuration' I will rename this as Configuration Properties, which might be less confusing since everything beneath this is an option. However, if you strongly disagree, let me know